lib/segment/tests/integration/segment_tests.rs

```diff
@@ -261,6 +261,7 @@ fn test_update_named_vector() {
         hnsw_ef: None,
         exact: true,
         quantization: None,
+        ignore_plain_index: false,
```

Comment:
I would suggest to think about a shorter name for this param. Ideally something within 2 words

Reply:
milvus have a similar parameter called ignore_growing. https://milvus.io/docs/search.md
We can either call it ignore_plain or ignore_growing?
wdyt? I have no strong preference

Reply:
I like `ignore_plain` better, 'growing' looks weird to me.

What about `ignore_unindexed`?

Reply:
also `indexed_only` might be an option

Reply:
indexed_only it is then